.github/workflows/deploy-preview-cleanup.yml

```
@@ -3,27 +3,127 @@ on:
+        id: destroy_app
+        continue-on-error: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.force_cleanup == 'true' }}
+        uses: superfly/fly-pr-review-apps@1.5.0
+        env:
+          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN_APP_REVIEW }}
+          FLY_REGION: ams
+          FLY_ORG: personal
+        with:
+          config: ./packages/app/fly.review.yaml
+          name: ${{ steps.vars.outputs.FLY_APP_NAME }}
+          
+                - name: 🗑 Delete deployment environment
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 71: mapping values are not allowed in this context

(syntax-check)